### PR TITLE
Use oks score for reprojections

### PR DESCRIPTION
**Pull request by roomrys**
### Description

This PR does 3 main things (described in more detail further down):
1. Use same OKS score for all `PredictedInstance.score`s
2. Add `InstanceGroup.score` attribute
3. Display `InstanceGroup.score` in table

Additionally, to use the `sleap.nn.evals.compute_oks` function inside a `sleap.io` file, we needed to move the `compute_oks` function to `sleap.util` to avoid a circular import between `sleap.nn` and `sleap.io` files.

#### Use same OKS score for all `PredictedInstance.score`s
Previously, all the `PredictedInstances` created and updated through (re)projection had a default nan score. This PR uses an average oks (calculated from ground truth `Instance`s and their reprojections) as the score for all (re)projected `PredictedInstance`s.

#### Add `InstanceGroup.score` attribute
To be most correct, even the `Instance`s which have reprojected nodes should have a score associated with them. There is currently no `Instance.score` attribute, and since this score really applies to the entire `InstanceGroup` (not just the `PredictedInstance`s in the `InstanceGroup`), this PR also adds an `InstanceGroup.score` attribute that is updated each time we call `InstanceGroup.update_points`.

#### Display `InstanceGroup.score` in table
This `InstanceGroup.score` is likely the most useful information for users to see, so this PR also adds a score column to the table displaying InstanceGroup information.

### Types of changes

- [ ] Bugfix
- [x] New feature
- [ ] Refactor / Code style update (no logical changes)
- [ ] Build / CI changes
- [ ] Documentation Update
- [ ] Other (explain)

### Does this address any currently open issues?
[list open issues here]

### Outside contributors checklist

- [ ] Review the [guidelines for contributing](https://github.com/talmolab/sleap/blob/develop/docs/CONTRIBUTING.md) to this repository
- [ ] Read and sign the [CLA](https://github.com/talmolab/sleap/blob/develop/sleap-cla.pdf) and add yourself to the [authors list](https://github.com/talmolab/sleap/blob/develop/AUTHORS)
- [ ] Make sure you are making a pull request against the **develop** branch (not *main*). Also you should start *your branch* off *develop*
- [ ] Add tests that prove your fix is effective or that your feature works
- [ ] Add necessary documentation (if appropriate)

#### Thank you for contributing to SLEAP!
:heart:

### CODE REVIEW

File: sleap/io/cameras.py [hunk 1]
```diff
@@ -1676,19 +1746,17 @@ def numpy(
                         f"{self.instance_groups}"
                     )
 
-        instance_group_numpys: List[np.ndarray] = []  # len(T) M=all x N x 2
+        instance_group_numpys: List[np.ndarray] = []  # len(T) M=include x N x 2
```
Comment (eberrigan) [inline sleap/io/cameras.py:1749]: is the comment right here?

File: tests/io/test_cameras.py [hunk 2]
```diff
@@ -590,6 +606,20 @@ def test_instance_group(
...
+        score=0.5,
+    )
+    assert instance_group_from_dict.score == 0.5
+    # The score of instances will NOT be updated on initialization.
+    for instance in instance_group_from_dict.instances:
+        if isinstance(instance, PredictedInstance):
+            assert instance.score != instance_group_from_dict.score
 
     # Test `__repr__`
     print(instance_group)
```
Comment (eberrigan) [inline tests/io/test_cameras.py:625]: did you mean to leave this?

**Review by eberrigan — APPROVED** [review]
Looks good! Just made some comments about a print statement and cleaning up a comment

File: tests/io/test_cameras.py (hunk 2 above)
Comment (roomrys) [reply tests/io/test_cameras.py:625]: Yes, the `__repr__` method is used when calling `print`

File: sleap/io/cameras.py (hunk 1 above)
Comment (roomrys) [reply sleap/io/cameras.py:1778]: Yes, in the for loop below, we add `T` elements to the `instance_group_numpys` list each with shape `M=include x N x 2`

**Pull request merged**